FogCreek/Glitch-Community
# Updated Collection Reordering Instruction UI

## Links
* Remix link: https://whispering-handball.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/43

## Changes:
* There's a hint section in the Collection Container that appears for collection owners
* You can click the Learn More button to expand and view the full instructions for reordering projects
* The hint section responds to changes to the collection color (changes font and background color)
* I also added a drop shadow to the elements when they're dragged (I wanted to add a slight rotation too, but it looks like react-sortable-hoc takes over the transform when moving, and I wasn't able to override).

## GIF/Screenshots:
![expand-hint](https://user-images.githubusercontent.com/519336/63129719-4c694100-bf86-11e9-8515-b3dbc4b94e86.gif)
![color](https://user-images.githubusercontent.com/519336/63129720-4f643180-bf86-11e9-91a6-af02fd7eda03.gif)
![box-shadow](https://user-images.githubusercontent.com/519336/63129729-512df500-bf86-11e9-84c9-c23b2f332162.gif)

## How To Test:
* Log into whispering-handball and go to one of your collections with more than 1 project.
* You should now see the reorder instructions tip and can click Learn More to expand to view the hint
* You should see a slight dropshadow when moving projects within the collection
